gadget/device_linux.go

```diff
@@ -138,3 +190,61 @@ func FindMountPointForStructure(ps *PositionedStructure) (string, error) {
 
 	return mountPoint, nil
 }
+
+func isWritableMount(entry *osutil.MountInfoEntry) bool {
+	// example mountinfo entry:
+	// 26 27 8:3 / /writable rw,relatime shared:7 - ext4 /dev/sda3 rw,data=ordered
+	return entry.Root == "/" && entry.MountDir == "/writable" && entry.FsType == "ext4"
```

Comment:
Silly question, do we always assume ext4 for /writable?

Reply:
That's how the fs is created in all images.